pom.xml

```diff
@@ -3,21 +3,48 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
-
+  <licenses>
+    <license>
+       <name>MIT License</name>
+       <url>https://opensource.org/license/MIT</url>
+    </license>
+  </licenses>
+  <organization>
+    <name>Eclipse Foundation</name>
+    <url>https://www.eclipse.org/</url>
+  </organization>
+  <issueManagement>
+    <url>https://github.com/eclipse-tracecompass/trace-event-logger/issues</url>
+    <system>GitHub Issues</system>
+  </issueManagement>
+  <scm>
+    <url>https://github.com/eclipse-tracecompass/trace-event-logger</url>
+    <connection>git@github.com:eclipse-tracecompass/trace-event-logger.git</connection>
+  </scm>
+  <developers>
+    <developer>
+      <name>Eclipse Trace Compass Community</name>
+      <organizationUrl>https://projects.eclipse.org/projects/tools.tracecompass</organizationUrl>
```

Comment:
Add also the organization under developer:
```xml
<organization>Eclipse Trace Compass Project</organization>
```

Reply:
Oops! thanks!